yarn.lock

```diff
@@ -385,16 +385,18 @@
     global-agent "^2.0.2"
     global-tunnel-ng "^2.7.1"
 
-"@electron/universal@1.0.5":
```

Comment:
all these changes are expected? (Just checking)

Reply:
I guess so. Installed new version through `yarn upgrade electron-builder@23.0.3`. Yarn made all of the changes in `yarn.lock`

Reply:
I guess you should ask `yarn` :D afaik it's auto-result from `yarn add electron-builder`. cc @aleksfront